app/src/main/res/values/strings.xml

```diff
@@ -18,16 +18,16 @@
     <string name="search_label">Search Publications</string>
     <string name="pf_placeholder">Publications Fragment</string>
     <string name="hello_blank_fragment">Hello blank fragment</string>
-    <string name="query_hint">Search by title...</string>
+    <string name="query_hint">Search by mTitle...</string>
```

Comment:
Haha, I think you did a `Refactor and rename` and it changed every `title` in the app to `mTitle`. You should fix these to they are `title`